### PR TITLE
Add missing assets in order to make Storybook work

### PR DESCRIPTION
**Pull request by zzarcon**
https://majames.github.io/react-audio-vis 💫

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,14 +17,21 @@
...
   "scripts": {
-    "build": "./node_modules/.bin/tsc",
+    "build": "tsc",
     "test": "react-scripts-ts test --env=jsdom",
     "storybook": "start-storybook -p 6006 -s ./stories/assets",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook -s ./stories/assets",
+    "deploy-storybook": "storybook-to-ghpages"
+  },
+  "storybook-deployer": {
```
Comment (majames) [inline package.json:34]: Do we need `storybook-deployer`? According to the docs it's the default: https://github.com/storybooks/storybook-deployer

File: package.json [hunk 2]
```diff
@@ -17,14 +17,21 @@
...
     "styled-components": "^1.4.6",
     "typescript": "^2.3.2"
   },
   "scripts": {
-    "build": "./node_modules/.bin/tsc",
+    "build": "tsc",
     "test": "react-scripts-ts test --env=jsdom",
     "storybook": "start-storybook -p 6006 -s ./stories/assets",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook -s ./stories/assets",
```
Comment (majames) [inline package.json]: do we need to add `-o .out` to the end of this command? https://github.com/storybooks/storybook-deployer

File: package.json (hunk 1 above)
Comment (zzarcon) [reply package.json:34]: Just added the ```[skip ci]``` to the commit message in order to make the process faster as we don't need that steep right?

File: package.json (hunk 2 above)
Comment (zzarcon) [reply package.json]: According to the docs yes, but looks like it works well now :/

**Comment by majames** [comment]
Did you want to add the required Travis config to get this happening? 😄 https://github.com/majames/react-audio-vis/issues/13

File: package.json (hunk 1 above)
Comment (majames) [reply package.json:34]: 👍

**Review by majames — DISMISSED** [review]

**Comment by zzarcon** [comment]
We can do #13 here yeah, do you know how to do it? Otherwise we can just delay it

**Comment by zzarcon** [comment]
@majames any idea of why build is failing?

**Comment by zzarcon** [comment]
Looks like it was a random build failure, @majames you happy with merging this?

**Review by majames — APPROVED** [review]

**Comment by majames** [comment]
Approved!

Failure was probs my bad dude, when I updated travis and/or coveralls config.

**Pull request merged**